packages/create-toolpad-app/index.ts

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env node
+
+import chalk from 'chalk';
+import { execaCommand } from 'execa';
+import inquirer from 'inquirer';
+import * as fs from 'fs/promises';
+import path from 'path';
+import { getPackageManager } from './utils/getPackageManager';
+import { isFolderEmpty } from './utils/isFolderEmpty';
+// Detect the package manager
+const packageManager = getPackageManager();
+
+// Install the dependencies
+const installDeps = async () => {
+  // eslint-disable-next-line no-console
+  console.log(`${chalk.blue('info')} - Installing dependencies`);
+
+  const installVerb = packageManager === 'yarn' ? 'add' : 'install';
+  const command = `${packageManager} ${installVerb} @mui/toolpad @mui/toolpad-core`;
+  await execaCommand(command, { stdio: 'inherit' });
+
+  // eslint-disable-next-line no-console
+  console.log(`${chalk.green('success')} - Dependencies installed successfully!`);
+};
+
+// Create a new directory and initialize a new project
+const scaffoldProject = async () => {
+  const name = await inquirer.prompt([
+    {
+      type: 'input',
+      name: 'projectName',
+      message: 'Enter the name of your project:',
+      default: 'my-toolpad-app',
+    },
+  ]);
+
+  // eslint-disable-next-line no-console
+  console.log(`Creating a new MUI Toolpad project in ${chalk.blue(name.projectName)}`);
+  try {
+    await fs.mkdir(name.projectName);
+    process.chdir(name.projectName);
```

Comment:
```suggestion
```
Can this be avoided? e.g. by passing `cwd` to `execa` instead. I feel like this could lead to weird issues where half the program is working in a certain working dir, and the other half in another. Moving code around could become awkward